### PR TITLE
Remove Deprecated Methods and Properties

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - Remove BTDropInBaseViewController showLoadingScreen:animated: method
 - Remove BTDropInRequest amount property
 - Remove BTUIKAppearance darkTheme and lightTheme methods

 ### Checklist

 - [x] Added a changelog entry

- @sestevens @sshropshire @scannillo

### CODE REVIEW

File: BraintreeDropIn/Public/BTDropInRequest.h [hunk 1]
```diff
@@ -12,13 +12,6 @@ typedef NS_ENUM(NSInteger, BTFormFieldSetting) {
 
 @interface BTDropInRequest : NSObject <NSCopying>
```
Comment (scannillo) [inline BraintreeDropIn/Public/BTDropInRequest.h]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Braintree iOS Drop-in SDK - Release Notes
 
+## unreleased
+
+* Remove deprecated methods and properties
+  * Remove BTDropInBaseViewController showLoadingScreen:animated: method
+  * Remove BTDropInRequest amount property
+  * Remove BTUIKAppearance darkTheme and lightTheme methods
```
Comment (scannillo) [inline CHANGELOG.md]: Can we add (NEXT_MAJOR_VERSION) next to each of these lines, so we don't accidentally release them in a minor version.

File: UnitTests/BTDropInControllerTests.swift [hunk 3]
```diff
@@ -1,55 +0,0 @@
...
-        dropInRequest.threeDSecureVerification = true
-        
-        let dropInController = BTDropInController(authorization: "development_tokenization_key",
-                                                  request: dropInRequest,
-                                                  handler: nil)
-        
-        XCTAssertEqual(dropInController?.dropInRequest.threeDSecureRequest?.amount, NSDecimalNumber(string: "12.54"))
-    }
-    
-    func testInitWithAuthorization_doesNotCreateThreeDSecureRequest_whenThreeDSecureIsNotRequested() {
```
Comment (scannillo) [inline UnitTests/BTDropInControllerTests.swift]: Why are we removing this test? Isn't this testing `dropInRequest.threeDSecureVerification = false` and not `dropInRequest.amount`.

Comment (scannillo) [reply UnitTests/BTDropInControllerTests.swift]: Nevermind. Makes sense to get rid of this file!

File: .travis.yml [hunk 4]
```diff
@@ -1,26 +1,29 @@
...
+#  - brew update || brew update
+#
+#install:
+#  - bundle install
+#  - bundle exec pod repo update --silent
+#  - bundle exec pod install
+#
+#script:
+#  - travis_wait bundle exec rake spec
+#
```
Comment (scannillo) [inline .travis.yml]: I'm a little confused why we left this commented out code? Can we add a comment to describe why it is there maybe?

Comment (sestevens) [reply .travis.yml]: Oh shoot, we can probably revert this. Steven and I were trying to debug why Travis kept failing, but we never resolved it. 

As a side note, I was thinking we could try [GitHub Actions](https://github.com/features/actions) as an alternative to Travis. Could be a good project for Open Dev.

Comment (scannillo) [reply .travis.yml]: Looks like travis build can't complete now

Comment (sestevens) [reply .travis.yml]: Yeah, it hasn't been working for a while :(

**Pull request merged**